packages/renderer/src/stores/image-checker-providers.ts

```diff
@@ -0,0 +1,51 @@
+/**********************************************************************
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import { writable, type Writable } from 'svelte/store';
+import type { ImageCheckerInfo } from '../../../main/src/plugin/api/image-checker-info';
+import { EventStore } from './event-store';
+
+const windowEvents = ['image-checker-provider-update'];
```

Comment:
I wonder if there should be an event for when an extension is removed.
If i uninstall the image-checker extension, do the registered imageCheckers get removed?

Reply:
Good catch! Without `image-checker-provider-remove` here, when disabling the image checker, the Check tab was still displayed.

When I set `image-checker-provider-remove` in the list, the Check tab disappears when the user disables the Image Checker, and re-appears when the extension is restarted.